forms-flow-bpm/src/main/java/org/camunda/bpm/extension/commons/io/event/CamundaEventListener.java

```diff
@@ -50,10 +59,18 @@ public void onTaskEventListener(DelegateTask taskDelegate) {
         try {
             if (isRegisteredEvent(taskDelegate.getEventName())) {
                 if (isAllowed("TASK_EVENT_DETAILS")) {
-                    this.template.convertAndSend("/topic/task-event-details", bpmObjectMapper.writeValueAsString(getTaskMessage(taskDelegate)));
+                    if(redisEnabled){
```

Comment:
move if(redisEnabled) to outer loop to avoid duplicate checks

Reply:
updated